extensions/adobe/experience/campaign/profile.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "ExperienceEvent Extension for Adobe Campaign",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "All ExperienceEvent properties specific to Adobe Campaign.",
+  "definitions": {
+    "profile": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/profile/timeZone": {
```

Comment:
@cdegroot-adobe @trieloff Should this be part of the core Profile?
@aarjain `timezone` and a better description will help.

Reply:
Added changes to incorporate timeZone into core profile XDM.

Reply:
@cdegroot-adobe  : Please check the addition of "timeZone" in standard profile in the new commit and share your comments on the same.